lib/assert.js

```diff
@@ -299,7 +295,7 @@ function expectedException(actual, expected) {
     return false;
   }
 
-  if (Object.prototype.toString.call(expected) == '[object RegExp]') {
+  if (Object.prototype.toString.call(expected) === '[object RegExp]') {
```

Comment:
Could we import `objectToString` from `internal/util` at the top and just call it here? And if so, would that be more readable?